.github/workflows/pod-level-pipeline.yml

```diff
@@ -116,61 +165,241 @@ jobs:
+        run: |
+          curl -sfL https://get.k3s.io | sh -s - --docker --write-kubeconfig-mode 664
+          kubectl wait node --all --for condition=ready --timeout=90s
+          kubectl get nodes
+          
+      - name: Litmus Infra Setup In Cluster-1
+        if: ${{ always() }}      
+        run: |
+          make build-litmus
+
+      - name: Deploy App In Cluster-1
+        if: ${{ always() }}      
+        run: make app-deploy
+    
+      - name: TCID-EC2-GENERIC-APP-POD-DELETE
+        if: ${{ always() }}      
+        run: make pod-delete
+        
+      - name: "[Debug]: check chaos resources"
+        if: ${{ failure() }}
+        continue-on-error: true
+        run: |
+          bash <(curl -s https://raw.githubusercontent.com/litmuschaos/litmus-e2e/master/build/debug.sh)
+          
+      - name: "[Debug]: check operator logs"
+        if: ${{ failure() }}
+        continue-on-error: true
+        run: |      
+          operator_name=$(kubectl get pods -n litmus -l app.kubernetes.io/component=operator --no-headers | awk '{print$1}')
+          kubectl logs $operator_name -n litmus > logs.txt
+          cat logs.txt        
+        
+      - name: Application Cleanup
+        if: ${{ always() }}      
+        run: make app-cleanup
+
+      - name: Litmus Cleanup
+        if: ${{ always() }}      
+        run: make litmus-cleanup      
+        
+      - name: Deleting K3S cluster
+        if: always()
+        run: /usr/local/bin/k3s-uninstall.sh        
 
-      - name: TCID-EC2-GENERIC-APP-DISK-FILL
-        if: ${{ always() }}
-        run: make disk-fill
 
   ### Runing Experiment Tunables
-  Experiment_Tunables:
-    needs: Setup_App_Deployment
+  PAP_100_Mode_Parallel:
     if: always()
-    container:
-      image: "${{ github.event.inputs.e2eTestImage }}"
-      volumes:
-        - /home/ubuntu/.kube:/root/.kube
-        - /home/ubuntu/.aws:/root/.aws
-        - /etc/kubernetes:/etc/kubernetes
-      env:
-        GO_EXPERIMENT_IMAGE: "${{ github.event.inputs.goExperimentImage }}"
-        LIB_IMAGE: "${{ github.event.inputs.libImage }}"
-        EXPERIMENT_IMAGE_PULL_POLICY: "${{ github.event.inputs.experimentImagePullPolicy }}"
-        CHAOS_NAMESPACE: "${{ github.event.inputs.expTunnableNS }}"
-        CHAOS_SERVICE_ACCOUNT: "${{ github.event.inputs.chaosServiceAccount }}"
-        APP_NS: "${{ github.event.inputs.expTunnableNS }}"
-        APP_LABEL: "app=nginx"
-        UPDATE_WEBSITE: "false"
-        KUBECONFIG: /root/.kube/config
-
-    runs-on: [self-hosted, pod-level-exp-tunable]
+    runs-on: ubuntu-latest
+    env:
+      GO_EXPERIMENT_IMAGE: "${{ github.event.inputs.goExperimentImage }}"
+      LIB_IMAGE: "${{ github.event.inputs.libImage }}"
+      EXPERIMENT_IMAGE_PULL_POLICY: "${{ github.event.inputs.experimentImagePullPolicy }}"
+      CHAOS_NAMESPACE: "${{ github.event.inputs.expTunnableNS }}"
+      CHAOS_SERVICE_ACCOUNT: "${{ github.event.inputs.chaosServiceAccount }}"
```

Comment:
Same as above

Reply:
PTAL `EXPERIMENT_IMAGE_PULL_POLICY: "${{ github.event.inputs.experimentImagePullPolicy }}"`, This is also not present in inputs array. Check in other places too.